docs/liquidators/manual.md

```diff
@@ -0,0 +1,162 @@
+
+Using the CLI, it is `umeed q leverage liquidation-targets`
+
+### Enabling the Query
+
+For performance reasons, this query is disabled by default:
+
+> Error: rpc error: code = Unknown desc = node has disabled liquidator queries
+
+Because this particular query iterates over all borrower accounts on the chain, it should not be enabled on validators or nodes supporting important infrastructure like block explorers.
+
+To enable the liquidation targets query on a node you control, start the node with the `-l` flag:
+
+`umeed start -l`
+
+All future examples were obtained using the CLI against a local umee node with the query enabled.
+
+### Using the Query
+
+When there are no borrowers eligible for liquidation, the return is empty:
+
+```sh
+% umeed q leverage liquidation-targets
+targets: []
+```
+
+If targets are available, they will appear as addresses
+
+```sh
+% umeed q leverage liquidation-targets
+targets:
+- umee1ycgwkvza827rvx7lllsv93ysxegxsky4jeuayt
+- umee1l2jv2mym7xd442cmeqka9yvd7vxelsplnn2qn8
+```
+
+### Choosing a Target
+
+Select one address and use leverage module queries to learn more about their position
+
+```sh
+% umeed q leverage account-summary umee1l2jv2mym7xd442cmeqka9yvd7vxelsplnn2qn8
+borrow_limit: "0.862311887319476919"
+borrowed_value: "0.981963999713893304"
+collateral_value: "1.077889859149346149"
+liquidation_threshold: "0.916206380276944227"
+supplied_value: "1.077889859149346149"
+```
+
+The values above indicate that the borrower is small, with borrowed and supplied values of approximately 1 USD. Their borrowed value (`$0.98`) is higher than their liquidation threshold (`$0.91`) so they can be liquidated.
+
+Next we want to see what tokens they have borrowed, and what collateral tokens of theirs can be claimed as a reward.
+
+```sh
+% umeed q leverage account-balances umee1l2jv2mym7xd442cmeqka9yvd7vxelsplnn2qn8
+borrowed:
+- amount: "83568"
+  denom: ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9
+collateral:
+- amount: "1073682"
+  denom: u/ibc/49788C29CD84E08D25CA7BE960BC1F61E88FEFC6333F58557D236D693398466A
+supplied:
+- amount: "1077906"
+  denom: ibc/49788C29CD84E08D25CA7BE960BC1F61E88FEFC6333F58557D236D693398466A
+```
+
+Here (and with the help of `umeed q leverage registered-tokens` to interpret ibc denoms) we see that the user has borrowed `0.083 ATOM` with collateral of `1.077 USDC`.
+
+If the borrower had multiple tokens collateralized or borrowed, this would be the time to choose which ones to repay and receive as a reward. In this case, the only option for the liquidator is to repay `ATOM` and receive `USDC`.
+
+## Executing a Liquidation Transaction
+
+Our intent is to repay `0.083 ATOM` on the borrower's behalf, and receive `USDC` in return. However, our liquidator account doesn't currently have `ATOM` on the Umee chain - it only has collateralized `USDC` and some `UMEE` left over for gas.
```

Comment:
let's add a note and example about the maximum repay. Position may not be eligible for full liquidation. Let's explain that, and provide a way how to calculate the maximum repay.

Reply:
Added